app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -122,55 +208,81 @@ def ems_clusters(extra_attributes = {})
     def storages(extra_attributes = {})
       attributes = {
         :model_class => ::Storage,
-        :association => :storages,
+        # TODO: change :manager_ref => [:location],
+        :association => :storages
```

Comment:
I think we'll need to look these up not by their association because we can link storages across EMSs